sonarqube/resource_sonarqube_group_member.go

```diff
@@ -185,15 +186,15 @@ func checkGroupMemberExists(groupName string, loginName string, m interface{}) (
 		"checkGroupMemberExists",
 	)
 	if err != nil {
-		return false, fmt.Errorf("error reading Sonarqube group members: %+v", err)
+		return fmt.Errorf("error reading Sonarqube group members for group '%s': %w", groupName, err)
```

Comment:
sorry, a copy-paste error

```suggestion
		return false, fmt.Errorf("error reading Sonarqube group members for group '%s': %w", groupName, err)
```

Reply:
Already fixed :-)